meta-resin-common/recipes-connectivity/networkmanager/resin-files/99dhcp_ntp

```diff
@@ -10,6 +10,9 @@ case "$action" in
 	# Run resin-ntp-config to update chronyc from ntp servers in config.json
 	/usr/bin/resin-ntp-config
 
+	# Inform chrony we are online
+	/usr/bin/chronyc online
```

Comment:
@ZubairLK I see that chrony has `onoffline` command as well, that might be interesting to use, when network changes occur? Just a thought, not entirely sure of the applicability and consequences. https://chrony.tuxfamily.org/doc/3.4/chronyc.html#onoffline

Reply:
`onoffline` is interesting. In our use case, I think `online` should work.